data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -198,9 +198,10 @@ A function may expect a function as a parameter, which is called a _higher-order
 - : int list = [0; 1; 4; 9; 16; 25]
 ```
 
-The name of this function begins with `List.` because it is part of the predefined library of functions acting on lists. This matter will be discussed more later. Function `List.map` takes two parameters: the second is a list, and the first is a function that can be applied to the list's elements, whatever they may be. `List.map` returns a list formed by applying the function provided as a parameter to each of the elements of the input list.
+The name of this function begins with `List.` because it is part of the predefined library of functions acting on lists. This matter will be discussed more later. Function `List.map` takes two arguments: the second is a list, and the first is a function that can be applied to the list's elements, whatever they may be. `List.map` returns a list formed by applying the function provided as argument to each of the elements of the input list.
```

Comment:
```suggestion
The name of this function begins with `List.` because it is part of the predefined library of functions acting on lists. This matter will be discussed more later. Function `List.map` has two parameters: the second is a list, and the first is a function that can be applied to the list's elements, whatever they may be. `List.map` returns a list formed by applying the function provided as argument to each of the elements of the input list.
```